.gitignore

```diff
@@ -0,0 +1,5 @@
+.dart_tool/
```

Comment:
I think there is no need to add this gitignore

Reply:
It was added automatically by IntelliJ. I think it's quite ok to have them ignore as you don't want this to be commited, but I'll remove